terraform/cloudwatch.tf.json.template.py

```diff
@@ -134,6 +134,52 @@
                     "dimensions": {
                         "HealthCheckId": "${aws_route53_health_check.composite-portal.id}",
                     }
+                },
+                "azul_fail_queue": {
+                    "alarm_name": f"{config.fail_queue_name}-message-count",
+                    "actions_enabled": True,
+                    "comparison_operator": "GreaterThanThreshold",
+                    "evaluation_periods": "1",
+                    "metric_name": "ApproximateNumberOfMessagesVisible",
+                    "namespace": "AWS/SQS",
+                    "period": "120",
+                    "statistic": "Average",
```

Comment:
Although this works, we can be more strict towards the information we are collecting. 
```suggestion
                    "statistic": "SampleCount",
```

Reply:
According to the [Amazon docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_concepts.html), `SampleCount` is just the number times a metric has been measured within a certain period. If you wanted a stricter statistic, wouldn't `Sum` be better?